ecs/elections/election_generator.py

```diff
@@ -0,0 +1,85 @@
+from random import gauss
+
+import names
+from django.db import transaction
+
+from ecs.elections.geo.models import Point
+from ecs.elections.models import Candidate, Preference
+from ecs.elections.models import Voter
+
+
+class ElectionGenerator(object):
+    election = None
+    candidates_mean_x = None
+    candidates_mean_y = None
+    candidates_sigma = None
+    voters_mean_x = None
+    voters_mean_y = None
+    voters_sigma = None
+    candidates_amount = None
```

Comment:
Zadawałem sobie to pytanie i doszedłem do wniosku takiego, że jeżeli miałyby się tu pojawić wartości domyślne, to znaczy że gdzieś po drodze nastąpił fakap. Więc lepiej żeby się wywaliło, niż po cichu zrobiło coś dziwnego, a potem byśmy się musieli zastanawiać co się stanęło :v